### PR TITLE
US-3497 US-3499 Adding support for is enabled and is selected on elements.

### PR DESCRIPTION
**Pull request by HeloiseRD**

### CODE REVIEW

File: lib/commands/element.js [hunk 1]
```diff
@@ -93,4 +93,62 @@ commands.getText = async function (el) {
 
 };
 
+commands.elementEnabled = async function (el) {
+
+  logger.debug("starting elementEnabled");
```
Comment (pfoster-youitv) [inline lib/commands/element.js]: I believe we should clean up some of these debug messages when we push this up.

File: lib/commands/element.js [hunk 2]
```diff
@@ -93,4 +93,62 @@ commands.getText = async function (el) {
...
+  let result;
+  try {
+    result = JSON.parse(data);
+  } catch (e) {
+    throw new Error("Bad response from elementSelected");
+  }
+
+  // get status returned
+  if (result.status == youiEngineDriverReturnValues.WEBDRIVER_STALE_ELEMENT)
+    throw new errors.StaleElementReferenceError();
```
Comment (pfoster-youitv) [inline lib/commands/element.js]: Any other errors we should throw?

**Comment by pfoster-youitv** [comment]
Looks good. Just reduce the debug logging messages and make sure we're throwing the right errors instead of a stale object reference for all the errors :)

**Comment by philgreenyouilabs** [comment]
please use 
https://youilabs.atlassian.net/browse/US-3497
https://youilabs.atlassian.net/browse/US-3499
thanks!

**Pull request merged**